imod/formats/prj/prj.py

```diff
@@ -547,6 +547,15 @@ def _create_dataarray_from_paths(
         _read=imod.idf._read,
         headers=headers,
     )
+
+    # remove coordinates in factor and addition but not in da
+    if type(factor) is xr.DataArray:
+        for d in factor.dims:
+            if d not in da.dims:
```

Comment:
You can probably do this at once with a set:

```python
missing_dims = set(factors.dims) - da.dims
if missing_dims:
     factor = factor.isel({d: 0 for d in missing_dims}, drop=True)
```

Reply:
done